### ISSUE TITLE
Radiogroup answer not selected when there is a second radiogroup with two visible-if's

### ISSUE THREAD

**Issue by leonalliander** [surveyjs/surveyjs#1396]
### Are you requesting a feature, reporting a bug or asking a question?

bug

### What is the current behavior?

With the code below, the visible answer is not selected.

### What is the expected behavior?

I expect the answer to be selected.

### How would you reproduce the current behavior (if this is a bug)?

I made a plunker that reproduces this behavior

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

https://plnkr.co/edit/qoKCmUBes5nprkvTD3Oy?p=preview

Test code

```
Survey
    .StylesManager
    .applyTheme("default");

var json = {
    "questions": [
        {
            "type": "radiogroup",
            "name": "group1",
            "valueName": "question1",
            "choices": [
                {
                    "value": "answer_possibility_1",
                    "visibleIf": "{other_question} = \"1\"" // If this line is removed value is selected
                },
                {
                    "value": "answer_possibility_2",
                    "visibleIf": "{other_question} = \"2\"" // If this line is removed value is selected
                }
            ],
            "visibleIf": "{another_question} = \"whatever\""
        },
        {
            "type": "radiogroup",
            "name": "group2",
            "valueName": "question1",
            "choices": [
                {
                    "value": "value_should_be_selected" // (but it is not)
                }
            ]
        }
    ]
};

window.survey = new Survey.Model(json);
window.survey.data = {
    "other_question": "2",
    "another_question": "blah",
    "question1": "value_should_be_selected"
};

survey
    .onComplete
    .add(function (result) {
        document
            .querySelector('#surveyResult')
            .innerHTML = "result: " + JSON.stringify(result.data);
    });

$("#surveyElement").Survey({model: survey});


```

### Specify your

* browser: chrome (latest)
* browser version:
* surveyjs platform (angular or react or jquery or knockout or vue): jquery 
* surveyjs version: 1.50

**Comment by tsv2013** [comment]
See https://surveyjs.answerdesk.io/ticket/details/T1239

**Comment by tsv2013** [comment]
The bug has been fixed with this commit: https://github.com/surveyjs/surveyjs/commit/36aae56d98449c90de7fbaf570b8a5dcaf2b700d
Released in the v1.0.52
